raft/raft.go

```diff
@@ -0,0 +1,403 @@
+package raft
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/gatewayd-io/gatewayd/config"
+	pb "github.com/gatewayd-io/gatewayd/raft/proto"
+	"github.com/hashicorp/raft"
+	raftboltdb "github.com/hashicorp/raft-boltdb"
+	"github.com/rs/zerolog"
+	"google.golang.org/grpc"
+)
+
+// Command types for Raft operations.
```

Comment:
I think the comment doesn't match the contents.

Reply:
Addressed in [d6f286b](https://github.com/gatewayd-io/gatewayd/pull/636/commits/d6f286b177eedc663e1fca7dda76fcb8c957f778)